magicgui/_tests/test_magicgui.py

```diff
@@ -66,6 +66,18 @@ def func(a: int = 1):
     assert gui.a == "1"
 
 
+def test_overriding_arg_display_name(qtbot):
+    """Test that a new label is given to a widget based on
+    the 'display_name' key."""
```

Comment:
```suggestion
    """Test that `display_name` overrides the label text."""
```

(fixes [linting error](https://github.com/napari/magicgui/pull/48/checks?check_run_id=1438037299#step:5:14))